### ISSUE TITLE
Gradle 6 support

### ISSUE THREAD

**Issue by h-ngo** [maiflai/gradle-scalatest#80]
Hi. Will there be support for Gradle 6 soon?

**Comment by maiflai** [comment]
> This plugin aims to be compatible with the current version of Gradle. The table below indicates the minimum required version.

I've bumped the version used to test the plugin and have not detected any issues under Java 8.

I see the travis build has failed under OpenJDK 11.

Do you have a specific issue that is preventing you from using Gradle 6 please?

**Comment by h-ngo** [comment]
This is false-positive. Closing this ticket.

Initially, we saw missing scala tests reported to sonar after upgrading gradle. We thought it might be related to gradle scalatest since it only states to support up to gradle 5.3. However, after taking a careful look, it's caused by the change in the xml report path due to gradle scoverage upgrade.

We are using Java 8 and Scala 2.12. Everything seems to work fine in Gradle 6 so far. Thanks for the quick response.
